examples/python-api/myownbloom.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python
+import khmer
+
+d1="ATGTACGGGCATTACGATTACCGATGTAG";
+d2="ATGACCAAACTCATTACGATTAGATATAG";
+
+ksize = 5
+target_table_size = 5e8
+num_tables = 4
+
+bf =khmer.Nodetable(ksize, target_table_size, num_tables);
+bf.consume(d1);
+
+cms=khmer.Counttable(ksize, target_table_size, num_tables);
+
+for kmer in cms.get_kmers(d2): #each kmer in d2
+	if bf.get(kmer) == 0:#if not in d1=
+		cms.consume(kmer)
+assert cms.get('CCAAA') 
```

Comment:
The `get` command returns an integer. This assert statement will fail if the `get` function returns a false value (0 in the case of integers) and will pass if it returns a non-false value (any other integer). I'd suggest being more explicit and showing exactly what you're testing for: `assert cmd.get('CCAAA') > 0`.